source/Octopus.Manager.Tentacle/Util/TelemetryService.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+using Octopus.Manager.Tentacle.Infrastructure;
+
+namespace Octopus.Manager.Tentacle.Util
+{
+    public interface ITelemetryService
+    {
+        Task<bool> SendTelemetryEvent(Uri serverUri, TelemetryEvent eventObj);
+    }
+    
+    public class TelemetryService: ITelemetryService
+    {
+        public async Task<bool> SendTelemetryEvent(Uri serverUri, TelemetryEvent eventObj)
+        {
+            using (var httpClient = new HttpClient())
```

Comment:
Do we need to do anything special here? E.g. to deal with proxies?

Reply:
Hmmm good point. The wizard does take proxy settings, so we should use them here. Nice one!